code/modules/maps/template_types/random_exoplanet/planet_types/chlorine.dm

```diff
@@ -23,6 +23,7 @@
 	exterior_atmos_temp            = null
 	level_generators               = list(
 		/datum/random_map/noise/exoplanet/chlorine,
+		/datum/random_map/noise/ore/poor,
```

Comment:
here and everywhere, wouldn't trailing comma result in a null entry in the lists?

Reply:
It had a trailing comma before. It just reduces line diff noise, I figured since it already had one I would keep it that way.